cron-backup.sh

```diff
@@ -3,15 +3,15 @@ set -e
 
 { echo ""; } | crontab -
 
-if [ -z "$SCHEDULE" ] 
+if [ -z "$SCHEDULE" ]
 then
 echo "SCHEDULE environment not defined! Using 0 0 * * *"
 SCHEDULE="0 0 * * *"
 fi
 
-if [ ! -z "$DIRS" ] 
+if [ ! -z "$DIRS" ]
 then
-  crontab -l | { cat; echo "$SCHEDULE bash $PWD/backup.sh"; } | crontab -
+  crontab -l | { cat; echo "$SCHEDULE bash backup.sh backup"; } | crontab -
```

Comment:
This was the problem that caused things to fail..otherwise nothing here should ahve caused issues.